File: app/src/main/java/com/song/deviceinfo/ui/bluetooth/BluetoothViewModel.java
package com.song.deviceinfo.ui.bluetooth;

import android.content.Context;

import com.song.deviceinfo.info.BluetoothInfo;
import com.song.deviceinfo.ui.base.NormalViewModel;

import java.util.List;

import androidx.core.util.Pair;

/**
 * Created by chensongsong on 2020/9/22.
 */
public class BluetoothViewModel extends NormalViewModel {

    public List<Pair<String, String>> getBluetoothInfo(Context context) {
        return BluetoothInfo.getBluetoothInfo(context);
    }

}

File: app/src/main/java/com/song/deviceinfo/ui/bluetooth/BluetoothViewModel.kt
package com.song.deviceinfo.ui.bluetooth

import android.content.Context
import androidx.core.util.Pair
import com.song.deviceinfo.info.BluetoothInfo
import com.song.deviceinfo.ui.base.NormalViewModel

/**
 * Created by chensongsong on 2020/9/22.
 */
class BluetoothViewModel : NormalViewModel() {
    fun getBluetoothInfo(context: Context): List<Pair<String, String>> {
        return BluetoothInfo.getBluetoothInfo(context)
    }
}
